validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -477,7 +477,7 @@ def validate_block(self, blkw, chain_head=None, chain=None):
 
         except Exception as e:
             LOGGER.exception(
-                "Unhandled exception BlockPublisher.validate_block()")
```

Comment:
`validate_block` is called in just two places, and both of those calls are wrapped in broad `except`s, so this catch isn't necessary at all.